assets/dev/js/frontend/tools/stretch-element.js

```diff
@@ -16,11 +17,11 @@ module.exports = elementorModules.ViewModule.extend( {
 	},
 
 	stretch() {
-		var containerSelector = this.getSettings( 'selectors.container' ),
+		var settings = this.getSettings(),
 			$container;
 
 		try {
-			$container = jQuery( containerSelector );
+			$container = jQuery( settings.selectors.container );
```

Comment:
Is this try in case jQuery is not loaded?

Reply:
I'm not sure, it was like that from before.

Reply:
understandable